echo-notifications/src/test/groovy/com/netflix/spinnaker/echo/notification/ManualJudgmentTemplateTest.groovy

```diff
@@ -17,6 +17,7 @@
 package com.netflix.spinnaker.echo.notification
 
 import com.netflix.spinnaker.echo.api.Notification
+import org.codehaus.groovy.runtime.StringGroovyMethods
```

Comment:
Confusing that IntelliJ thinks this isn't used, yet tests fail without it.

Reply:
Yeah it actually isn't - I actually just changed the entire thing to be a Java String block rather than relying on `stripIndent`. I'll fix it up.